.eslintrc.js

```diff
@@ -18,7 +18,10 @@ module.exports = {
         'plugin:n/recommended',
         'plugin:promise/recommended'
       ],
-      files: ['**/*.{cjs,js,mjs}'],
+      files: [
+        '**/*.{cjs,js,mjs}',
+        '**/*.md/*.{cjs,js,mjs}'
```

Comment:
**question**: That's an odd glob with `*.md` being a "directory" when they're actually files. What does it represent? Is it to match the codeblocks within markdown?

Reply:
Yeah it matches all `cjs`, `js` and `mjs` blocks in `*.md` files
https://github.com/eslint/eslint-plugin-markdown#advanced-configuration

Reply:
Could we add a little comment in the config to link us there so everyone can quickly find about it, please?
```suggestion
        '**/*.md/*.{cjs,js,mjs}' // https://github.com/eslint/eslint-plugin-markdown#advanced-configuration
```

Reply:
Are there any other plugin-related rules or settings that should have comments too?

Aware there's probably quite a few things that don't make immediate sense